workbench-core/accounts/src/utilities/hostingAccountLifecycleService.ts

```diff
@@ -479,6 +505,17 @@ export default class HostingAccountLifecycleService {
     return currOutputVal;
   }
 
+  private _getEnvironmentFilesPathForArn(artifactBucketArn: string): string {
+    const parsedBucketArn = artifactBucketArn.replace('arn:aws:s3::::', '').split('/');
+    const bucketName = parsedBucketArn[0];
+
+    if (_.isEmpty(bucketName) || bucketName.length === 0) {
+      throw Boom.internal(`Could not identify bucket name in S3 artifact bucket ARN ${artifactBucketArn}`);
```

Comment:
Nitpick. Let's not return the bucket arn since this is leaking implementation details to the customer. What we can do here is log the bucket name so the IT Admin can see the bucket name in the log, but not the customer. The customer won't be able to do anything with the bucket arn info anyway.

```
console.error(`Could not identify bucket name in S3 artifact bucket ARN ${artifactBucketArn}`)
throw Boom.internal('Could not identify bucket name in S3 artifact bucket');
```

Reply:
Updated.